run-tests-with-build-kogito-apps-parent/kogito-apps-project-sources-test/pom.xml

```diff
@@ -107,8 +107,6 @@
                 <pomInclude>**/pom.xml</pomInclude>
               </pomIncludes>
               <pomExcludes>
```

Comment:
you can even remove this `pomExcludes` node

Reply:
See main PR